complete/src/main/java/dto/TourGuide.java

```diff
@@ -0,0 +1,13 @@
+package TourGuide.dto;
+
+public class TourGuide {
+
+    private String KeyLocations;
+
+    public TourGuide(String keyLocations){
+        this.KeyLocations = keyLocations;
+    }
+
+    public String getKeyLocations() { return KeyLocations; }
```

Comment:
Tour guide that holds a key location.

Source - https://www.baeldung.com/java-clean-code